rustfmt-core/rustfmt-lib/src/test/mod.rs

```diff
@@ -21,21 +21,10 @@ const DIFF_CONTEXT_SIZE: usize = 3;
 
 // A list of files on which we want to skip testing.
 const SKIP_FILE_WHITE_LIST: &[&str] = &[
+    "issue-3253/paths",
```

Comment:
Placing a remaining whitelisted file up in lexicographical order after 'cleaning house'